### PR TITLE
Add disabled option for text area and input

### PR DESCRIPTION
**Pull request by matheusagcosta**

### CODE REVIEW

**Comment by kylefiedler** [comment]
:100:

File: app/assets/stylesheets/_forms.scss [hunk 1]
```diff
@@ -50,6 +50,11 @@ textarea {
     box-shadow: $form-box-shadow-focus;
     outline: none;
   }
+  
+  &:disabled {
+    background-color: darken($base-background-color, 6);
```
Comment (tysongach) [inline app/assets/stylesheets/_forms.scss]: `6` needs a `%`

**Comment by tysongach** [comment]
One minor comment, otherwise this is :+1:

**Comment by matheusagcosta** [comment]
@tysongach Fixed.

**Comment by robbyoconnor** [comment]
+1 -- I want to see this merged!

**Comment by whmii** [comment]
I could see adding the following to remove hover state.

```scss
&:hover {
  border: $base-border;
}
```

**Comment by matheusagcosta** [comment]
@whmii Added.

**Comment by tysongach** [comment]
Merged.

Thanks!

**Pull request closed without merging**